content/applications/general/email_communication/email_domain.rst

```diff
@@ -111,6 +99,11 @@ Conformance)` record of a domain name tell the destination server what to do wit
 that fails the :abbr:`SPF (Sender Policy Framework)` and/or :abbr:`DKIM (DomainKeys Identified
 Mail)` check.
 
+.. example::
+   DMARC: TXT record
+
+   v=DMARC1; p=none; sp=none; aspf=r; adkim=r; pct=100; rua=mailto:example@company.com;
```

Comment:
```suggestion
   v=DMARC1; p=none;
```

Let's simplify this at maximum.

This DMARC is enough for a first configuration (just won't do anything).
I would rather add a link to this page:
https://dmarc.org/overview/
Explaining which parameters can be added.

(This change is subject to discussion :)  -- maybe @jqu-odoo might have an opinion on it!)

Reply:
I've added this in as well, great suggestion!